x/oracle/internal/keeper/operator.go

```diff
@@ -67,6 +67,9 @@ func (k Keeper) CreateOperator(ctx sdk.Context, address sdk.AccAddress, collater
 	if k.IsBelowMinCollateral(ctx, collateral) {
 		return types.ErrNoEnoughCollateral
 	}
+	if !k.CertKeeper.IsCertified(ctx, "Address", address.String(), "ORACLEOPERATOR") {
+		return types.ErrNotCertified
+	}
```

Comment:
if this needs to be in for update 1 it needs a height check

Reply:
updated! Thanks